doc/QuickStart.md

```diff
@@ -29,23 +32,48 @@ Installing
 
 ### Prerequisites
 
-#### Installing CPAN modules
+#### Requirements *(root)*
+
+	$ dnf -y install perl perl-CPAN git screen crontabs nano
+> or 		$ aptitude install perl perl-CPAN git screen crontabs nano
+> or 		$ apt-get install perl perl-CPAN git screen crontabs nano
```

Comment:
Local operating system configuration is beyond the scope of this QuickStart guide. Users are expected to already know how to set up their local environment and any utilities they may want to use.

This QuickStart guide is for quickly getting started with PBot. The guide is not intended to provide support for a subset of package managers.